radialgraph/src/main/java/com/duartbreedt/radialgraph/drawable/RadialGraphDrawable.kt

```diff
@@ -31,6 +32,35 @@ class RadialGraphDrawable(
 
             sectionState.paint = buildPhasedPathPaint(sectionState)
             canvas.drawPath(sectionState.path!!, sectionState.paint!!)
+
+            if (graphConfig.graphNodeType != GraphNode.NONE) {
```

Comment:
I'd prefer if this check was specific to `GraphNode.PERCENT`. E.g.
```suggestion
            if (graphConfig.graphNodeType != GraphNode.PERCENT) {
```
If we add another enum then this code will run, which is not desired.